dhcp/dhcp.go

```diff
@@ -0,0 +1,202 @@
+package dhcp
```

Comment:
Packages must be preceded with a one line comment explaining the purpose of the package